### PR TITLE
Feature/india template

### PR DESCRIPTION
**Pull request by speppy**

### CODE REVIEW

**Comment by tomasmoose** [comment]
Another note, now when we have 4 themes the styles in the admin does not work correctly when selecting a theme, ( see screenshot )

screen shot 2018-09-03 at 12 05 46


I think that all you need to add [here](https://github.com/Vizzuality/forest-atlas-landscape-cms/blob/develop/app/assets/stylesheets/layouts/admin/l-site-creation.scss#L46) is: 

```
    align-items: center;
    justify-content: space-between;
    flex-direction: column;
    flex-flow: wrap;
```

And we can have 2 columns instead like this: 

screen shot 2018-09-03 at 12 06 57

**Comment by tomasmoose** [comment]
screen shot 2018-09-03 at 12 21 52


Looks like the footer is above the homepage content, I think the `no_hight` is the issue here, why do we have the `height: 0!important`? is it solving another issue?

File: lib/untitled [hunk 1]
```diff
@@ -0,0 +1,58 @@
+
```
Comment (davidsingal) [inline lib/untitled]: Could we put a name to this file? Are we using it?

Comment (speppy) [reply lib/untitled]: Deleted this file in the next commit.

File: config/database.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ default: &default
 
 development:
   <<: *default
-  database: <%= ENV.fetch("POSTGRES_DATABASE") { "falcms_dev" } %>
+  database: <%= ENV.fetch("POSTGRES_DATABASE") { "falcms_dev_prod" } %>
```
[the remark this answers is not in the document]
Comment (speppy) [reply config/database.yml]: Done. Pushed the change in next commit.

File: app/javascript/components/CoverPage.js [hunk 3]
```diff
@@ -19,13 +19,37 @@ class CoverPage extends React.Component {
...
+    let atlas_button;
+
+    if(siteTemplateName != 'INDIA'){
+      page_title = <h2 className="cover-title"> {page.content_type == 3 ? current.name : page.name} </h2>;
+      cover_subtitle = page.content_type !== 3 && <h2 className="cover-subtitle">{current.name}</h2>;
+    }else{
+      if(page.content_type == 3){
+        page_title = <h2 className="cover-title-left">{current.name} </h2>;
+        cover_description = <div className="cover-desc-wrapper" ><h2 className="cover-description">{page.description}</h2></div>;
+        atlas_button = <div class="button_wrapper" height="0px"> <a href="/atlas" className="home_page_atlas_button">Launch Atlas</a> </div>;
```
[the remark this answers is not in the document]
Comment (speppy) [reply app/javascript/components/CoverPage.js]: Fixed. Pushed in next commit.

File: app/javascript/components/CoverPage.js [hunk 4]
```diff
@@ -19,13 +19,37 @@ class CoverPage extends React.Component {
...
+    let page_title;
+    let cover_subtitle;
+    let cover_description_wrapper;
+    let cover_description;
+    let atlas_button;
+
+    if(siteTemplateName != 'INDIA'){
+      page_title = <h2 className="cover-title"> {page.content_type == 3 ? current.name : page.name} </h2>;
+      cover_subtitle = page.content_type !== 3 && <h2 className="cover-subtitle">{current.name}</h2>;
+    }else{
```
[the remark this answers is not in the document]
Comment (speppy) [reply app/javascript/components/CoverPage.js]: Have pushed a version. Please review it.

**Pull request merged**